openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-area.js

```diff
@@ -268,14 +277,6 @@ var DrawWbArea = function() {
 			, tcid = __getWbContentId(obj.wbId)
 			, wb = OmUtil.tmpl('#wb-area', tcid).attr('aria-labelledby', tid)
 			, li = OmUtil.tmpl('#wb-area-tab')
-				.contextmenu(function(e) {
```

Comment:
Moved to _addCloseBtn as otherwise contextmenu is applied on too many HTML elements. Like close button.